Cargo.toml

```diff
@@ -11,6 +11,7 @@ version = "0.8.0"
 
 [dependencies]
 dav1d-sys = { version = "0.5.0", path = "dav1d-sys" }
+libc = "0.2"
```

Comment:
Why is it needed?

Reply:
maybe it is nicer if you put it as dependency for dav1d and re-export from there.

Reply:
I agree, something like https://github.com/rust-av/dav1d-rs/pull/68#discussion_r988808325